test/rps-test.js

```diff
@@ -0,0 +1,43 @@
+import { getThrowFromNumber } from '../src/get-throw.js';
+
+const test = QUnit.test;
+
+test('2 is scissors', function(assert) {
```

Comment:
try to be a little more descriptive here, such as "returns scissors when given the number 2"